policies/testing.md

```diff
@@ -42,6 +42,50 @@ A test may be omitted where writing the test would result in disproportionately
 more effort than writing the code being tested. For example, difficult to
 reproduce error conditions.
 
+Triage Labels
+-------------
+
+Before approving a PR, the applicability of the testing policy to the PR must
+be assessed and an appropriate label applied to the PR. One of the following
+labels must be applied:
+
+- `tests: present`, to be added where suitable tests are included in the same PR.
+- `tests: needed`, where the testing policy requires tests but they are not
+  currently included in a PR.
+- `tests: exempted`, where tests are not included in a PR but are not required
+  by this policy (for example, because the PR concerns only documentation,
+  or because writing the test would result in disproportionately more effort;
+  see above).
+- `tests: deferred`, where the tests are required by the testing policy
+  but it is intended to add these tests in a subsequent PR.
+- `tests: added in subsequent PR`, where the tests were deferred to a subsequent
+  PR which has now been merged. This may be set on a given PR after it is
+  merged, replacing the `tests: deferred` label.
+
+The `tests: needed` label blocks merging of the PR and constitutes a hold.
+None of the other labels listed above block merging of a PR.
+
+Only one of the above labels should be set on a PR at a given time.
+
+It will generally be readily apparent which label is applicable
+at any given time. However, if there is a lack of agreement over the
+status of a PR in regards to the test policy, this should be resolved
+via the normal review process. For example, committers are not required
+to agree to use of the `tests: deferred` label if they do not feel
+it is appropriate.
+
+Use and approval of the `tests: deferred` label is at the discretion of
+committers. It is recommended that this label be used only when a subsequent PR
```

Comment:
Do you mean `openssl/committers` when you say "committers", or do you mean anyone who submits a PR?  For the latter, this is going to be problematic, as PR submitters that aren't part of a recognised team (`openssl/committers` for example) do not have the power to set any label, as far as I recall...

Reply:
The former. I thought this term was well understood. Will change it to `OpenSSL Committers` which is maybe a bit clearer.